src/main/app/controller/PostController.ts

```diff
@@ -78,6 +78,29 @@ export class PostController<T extends AnyObject> {
     });
   }
 
+  public async sendToApplicant2ForReview(req: AppRequest<T>, res: Response, formData: Partial<Case>): Promise<void> {
+    Object.assign(req.session.userCase, formData);
+    this.form.setFormState(req.session.userCase);
+    req.session.errors = this.form.getErrors(formData);
+
+    try {
+      const twoWeeks = 1000 * 60 * 60 * 24 * 14;
```

Comment:
```suggestion
      const twoWeeks = 14 * 24 * 60 * 60 * 1000;
```
just for constitency, not a big deal.

Reply:
sure